### PR TITLE
Rework boxconstruct stripped

### PR DESCRIPTION
**Pull request by mmatera**
This PR is the part of #181 that does the actual rework, without the comments in ``mathics.core``.

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/186"></a>

### CODE REVIEW

File: mathics/builtin/__init__.py [hunk 1]
```diff
@@ -216,3 +216,12 @@ def is_builtin(var):
 _builtins = {}
 
 add_builtins(new_builtins)
+
+
+display_operators_set = set()
```
Comment (TiagoCavalcante) [inline mathics/builtin/__init__.py:221]: I didn't find where it is used.

Comment (mmatera) [reply mathics/builtin/__init__.py:221]: This is used in `_BoxedString`

File: mathics/builtin/box/inout.py [hunk 2]
```diff
@@ -1,11 +1,179 @@
...
+                return render("<ms>%s</ms>", text[1:-1])
+            else:
+                outtext = ""
+                for line in text[1:-1].split("\n"):
+                    outtext += render("<mtext>%s</mtext>", line)
+                return outtext
+        elif text and ("0" <= text[0] <= "9" or text[0] == "."):
+            return render("<mn>%s</mn>", text)
+        else:
+            if text in operators or text in extra_operators:
```
Comment (mmatera) [inline mathics/builtin/box/inout.py:100]: Here...

File: mathics/builtin/__init__.py (hunk 1 above)
Comment (mmatera) [reply mathics/builtin/__init__.py:221]: In ``mathics.core.atom.String.boxes_to_mathml`` this set is built each time the method is called. I just move it to ``mathics.builtin.base`` to do it just once.

**Pull request closed without merging**